core-ui/src/components/Lambdas/constants.js

```diff
@@ -552,10 +552,10 @@ export const ENVIRONMENT_VARIABLES_PANEL = {
 export const FORMS = {
   RESOURCE_NAME: {
     LABEL: 'Name',
-    INLINE_HELP: `The name must consist of lower case alphanumeric characters or dashes, and must start and end with an alphanumeric character (e.g. 'my-name1').`,
+    INLINE_HELP: `The name must contain lower case alphanumeric characters, can contain '-'  (like 'my-name1').`,
```

Comment:
According to previous messages it should be
```suggestion
    INLINE_HELP: `Name must contain lower case alphanumeric characters, can contain '-'  (like 'my-name1').`,
```